monitor.go

```diff
@@ -491,14 +492,20 @@ func main() {
 func runMonitor() {
 	firstLoop := true
 	healthy = true
+	duration := 30 * time.Second
 	mainLoop := func() {
-		duration, err := time.ParseDuration(*testInterval)
+		userDuration, err := time.ParseDuration(*testInterval)
 		if err != nil {
 			log.Fatalf("Failed to parse time interval: %v", err)
 		}
 
 		for {
 			if !firstLoop {
+				if status == false {
```

Comment:
Be better to just leave the other assignments as-is, and only update the duration to the user duration where we previously set it to 2 minutes